.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,3 +1,9 @@
 #### Checklist
-- [ ] Unit tests
-- [ ] Updated testplan
+- [ ] Functionality meets specifications
+- [ ] Functionality working
+- [ ] Code reviewed
+- [ ] Code tested
+- [ ] User documentation updated
+- [ ] Test plan template updated
+- [ ] Clean commits
```

Comment:
@Mark-de-Haan: The PR maker should take care of the clean commits, but the reviewer should review... same as the rest of the review...